### PR TITLE
Add istio 1.11.8

### PR DESCRIPTION
**Pull request by doflamingo721**
Note: This PR contains a change in directory structure for istio charts.

### CODE REVIEW

File: packages/rancher-istio/1.12/rancher-tracing/charts/Chart.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ annotations:
   catalog.rancher.io/namespace: istio-system
   catalog.rancher.io/release-name: rancher-tracing
 apiVersion: v1
-appVersion: 1.33.0
+appVersion: 1.32.0
 description: A quick start Jaeger Tracing installation using the all-in-one demo.
   This is not production qualified. Refer to https://www.jaegertracing.io/ for details.
-name: rancher-tracing
```
Comment (aiyengar2) [inline packages/rancher-istio/1.12/rancher-tracing/charts/Chart.yaml]: why did we change the name?

File: packages/rancher-istio/1.12/rancher-kiali-server/generated-changes/patch/Chart.yaml.patch [hunk 2]
```diff
@@ -25,7 +25,7 @@
    name: Kiali
    url: https://kiali.io
 -name: kiali-server
-+name: rancher-kiali-server
```
Comment (aiyengar2) [inline packages/rancher-istio/1.12/rancher-kiali-server/generated-changes/patch/Chart.yaml.patch]: why did we change the name?

File: packages/rancher-istio/1.12/rancher-istio/charts/Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Istio
-  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.24.0-0'
```
Comment (aiyengar2) [inline packages/rancher-istio/1.12/rancher-istio/charts/Chart.yaml]: So we should only be able to install this chart in 1.19, 1.20, 1.21, and 1.22?

File: packages/rancher-istio/1.12/rancher-istio/charts/Chart.yaml [hunk 4]
```diff
@@ -11,14 +11,14 @@ annotations:
...
-appVersion: 1.13.3
+appVersion: 1.12.6
 description: A basic Istio setup that installs with the istioctl. Refer to https://istio.io/latest/
   for details.
 icon: https://charts.rancher.io/assets/logos/istio.svg
 keywords:
 - networking
 - infrastructure
 name: rancher-istio
-version: 1.13.3
```
Comment (aiyengar2) [inline packages/rancher-istio/1.12/rancher-istio/charts/Chart.yaml]: this version shouldn't be modified, it's done in the package.yaml

File: packages/rancher-istio/1.11/rancher-tracing/charts/values.yaml [hunk 5]
```diff
@@ -48,3 +48,4 @@ service:
   name: http-query
   type: ClusterIP
   externalPort: 16686
+
```
Comment (aiyengar2) [inline packages/rancher-istio/1.11/rancher-tracing/charts/values.yaml:51]: why did we add this?

File: packages/rancher-istio/1.11/rancher-kiali-server/generated-changes/patch/Chart.yaml.patch [hunk 6]
```diff
@@ -25,7 +25,7 @@
    name: Kiali
    url: https://kiali.io
 -name: kiali-server
-+name: rancher-kiali-server
```
Comment (aiyengar2) [inline packages/rancher-istio/1.11/rancher-kiali-server/generated-changes/patch/Chart.yaml.patch]: why did we change the name?

File: packages/rancher-istio/1.11/rancher-istio/charts/Chart.yaml [hunk 7]
```diff
@@ -1,7 +1,7 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Istio
-  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.24.0-0'
```
Comment (aiyengar2) [inline packages/rancher-istio/1.11/rancher-istio/charts/Chart.yaml]: So this should be supported in 1.19, 1.20, 1.21, and 1.22 only?

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

File: packages/rancher-istio/1.12/rancher-istio/charts/Chart.yaml (hunk 3 above)
Comment (samkulkarni20) [reply packages/rancher-istio/1.12/rancher-istio/charts/Chart.yaml]: Yes. That's correct. It's according to the istio support matrics: [https://istio.io/latest/docs/releases/supported-releases/](https://istio.io/latest/docs/releases/supported-releases/)

File: packages/rancher-istio/1.11/rancher-istio/charts/Chart.yaml (hunk 7 above)
Comment (doflamingo721) [reply packages/rancher-istio/1.11/rancher-istio/charts/Chart.yaml]: This should support from k8s 1.18 to 1.122 as per the [istio support](https://istio.io/latest/docs/releases/supported-releases/) matrix. Will update to include 1.18

File: packages/rancher-istio/1.11/rancher-kiali-server/generated-changes/patch/Chart.yaml.patch (hunk 6 above)
Comment (doflamingo721) [reply packages/rancher-istio/1.11/rancher-kiali-server/generated-changes/patch/Chart.yaml.patch]: Seems like a mistake while updating. Will revert the name to the `rancher-kiali-server`.

File: packages/rancher-istio/1.11/rancher-tracing/charts/values.yaml (hunk 5 above)
Comment (doflamingo721) [reply packages/rancher-istio/1.11/rancher-tracing/charts/values.yaml:51]: A new line got added at the end of the file. Will rectify this.

File: packages/rancher-istio/1.12/rancher-kiali-server/generated-changes/patch/Chart.yaml.patch (hunk 2 above)
Comment (doflamingo721) [reply packages/rancher-istio/1.12/rancher-kiali-server/generated-changes/patch/Chart.yaml.patch]: Seems like a mistake while updating. Will revert back to `rancher-kiali-server`.

File: packages/rancher-istio/1.12/rancher-tracing/charts/Chart.yaml (hunk 1 above)
Comment (doflamingo721) [reply packages/rancher-istio/1.12/rancher-tracing/charts/Chart.yaml]: Reverted the name to `rancher-tracing`

File: packages/rancher-istio/1.12/rancher-istio/charts/Chart.yaml (hunk 4 above)
Comment (doflamingo721) [reply packages/rancher-istio/1.12/rancher-istio/charts/Chart.yaml]: Added the appropriate version.

File: packages/rancher-istio/1.11/rancher-kiali-server/generated-changes/patch/templates/_helpers.tpl.patch [hunk 8]
```diff
@@ -17,13 +17,10 @@
...
-@@ -170,4 +177,27 @@
- {{- else }}
-   {{- .Release.Namespace }}
+@@ -141,3 +148,26 @@
+   {{- end }}
+ {{- end }}
  {{- end }}
--{{- end }}
-\ No newline at end of file
-+{{- end }}
```
[the remark this answers is not in the document]
Comment (aiyengar2) [reply packages/rancher-istio/1.11/rancher-kiali-server/generated-changes/patch/templates/_helpers.tpl.patch]: Talked offline:

> the change was not made manually, I updated the _helpers.tpl to match kiali 1.41. And then I did make patch  and this was the resulting patch.

File: packages/rancher-istio/1.11/rancher-istio/charts/Chart.yaml [hunk 9]
```diff
@@ -11,14 +11,14 @@ annotations:
...
+appVersion: 1.11.8
 description: A basic Istio setup that installs with the istioctl. Refer to https://istio.io/latest/
   for details.
 icon: https://charts.rancher.io/assets/logos/istio.svg
 keywords:
 - networking
 - infrastructure
 name: rancher-istio
-version: 1.13.3
+version: 100.1.3+up1.11.8
```
Comment (aiyengar2) [inline packages/rancher-istio/1.11/rancher-istio/charts/Chart.yaml:24]: this version seems incorrect, 100.1.3 should be omitted.

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

File: packages/rancher-istio/1.11/rancher-istio/charts/Chart.yaml (hunk 9 above)
Comment (doflamingo721) [reply packages/rancher-istio/1.11/rancher-istio/charts/Chart.yaml]: Updated the version.

**Review by aiyengar2 — DISMISSED** [review]
LGTM

**Review by aiyengar2 — APPROVED** [review]

**Pull request merged**